main.js

```diff
@@ -0,0 +1,101 @@
+/*eslint-disable */
```

Comment:
- [ ] Please avoid disabling all the linters, they ensure consistency in code format and force certain style guidelines that are important in order to have clean and performant code. Instead, you could only disable the rule that's causing problems, using: `/* eslint-disable rule-you-want-to-disable */`. Please consider fixing these disabling comments. 🤓 🛠️